### ISSUE TITLE
[BUG] Conventional Commits shows "Choose a repository" prompt every time

### ISSUE THREAD

**Issue by zerodayyy** [vivaxy/vscode-conventional-commits#289]
### Describe the bug

After upgrading to 1.24.4, the extension started to show a prompt "Choose a repository" on every action even when clicking the circle icon in "Source Control" panel:

<img width="167" alt="image" src="https://user-images.githubusercontent.com/49662783/207116152-0605d645-a747-47a7-980a-905ca9595cc8.png">

This is annoying, since the button is already located under a specific repo in UI.

### To reproduce

1. Install Conventional Commits v1.24.4
2. Open a Workspace with multiple Git-versioned folders
3. Open Source Control view in side panel
4. Click "Conventional Commits" circle icon
5. Repo prompt is displayed

### Expected behavior

Only commit message details are prompted

### Screenshots

<img width="601" alt="image" src="https://user-images.githubusercontent.com/49662783/207117025-58c29b72-4a6b-4fff-a922-2c033cf90e74.png">

### Application logs

<details>
<summary>Spoiler</summary>

```
[info] Conventional commits started.
[info] VSCode version: 1.74.0
[info] Git version: 1.0.0
[info] VSCode Conventional Commits version: 1.24.4
[info] conventionalCommits.autoCommit: true
[info] conventionalCommits.silentAutoCommit: false
[info] conventionalCommits.emojiFormat: code
[info] conventionalCommits.gitmoji: true
[info] conventionalCommits.lineBreak: 
[info] conventionalCommits.promptBody: true
[info] conventionalCommits.promptCI: false
[info] conventionalCommits.promptFooter: true
[info] conventionalCommits.promptScopes: true
[info] conventionalCommits.scopes: redacted-1,redacted-2,redacted-3
[info] conventionalCommits.showEditor: false
[info] conventionalCommits.showNewVersionNotes: false
[info] conventionalCommits.editor.keepAfterSave: false
[info] git.enableSmartCommit: true
[info] git.smartCommitChanges: all
[info] git.postCommitCommand: none
[info] arg: undefined
[info] git.repositories: /Users/zerodayyy/Redacted/redacted-repo-1, /Users/zerodayyy/Redacted/redacted-repo-2
[info] workspaceFolders: /Users/zerodayyy/Redacted/redacted-repo-1, /Users/zerodayyy/Redacted/redacted-repo-2
```
</details>

**Comment by yi-Xu-0100** [comment]
👋 @zerodayyy, thanks for your report. 😁

By the [official reply](https://github.com/microsoft/vscode/issues/168758#issuecomment-1346085303), this is not a long-term fix, I will keep an eye on this issue and hopefully fix it permanently.

Now you can try the [beta extension](https://github.com/vivaxy/vscode-conventional-commits/actions/runs/3682169372#artifacts), please let me know if he still has other problems. 😊

**Comment by zerodayyy** [comment]
Thanks, beta extension seems to work properly

**Comment by yi-Xu-0100** [comment]
👋 @zerodayyy, I am trying to figure out a long-term fix for this issue, and by referencing a similar type of VS Code extension ([Git History](https://github.com/DonJayamanne/gitHistoryVSCode)), I resubmitted a [PR#292](#292) and generated a new beta extension ([vscode-conventional-commits-ec78c36.vsix](https://github.com/vivaxy/vscode-conventional-commits/actions/runs/3762872047#artifacts)), hoping you can help and see if it has any other potential issues? 😁

**Comment by zerodayyy** [comment]
@yi-Xu-0100 for sure, already installed it. I'll let you know if there are any other issues with it

**Comment by yi-Xu-0100** [comment]
@zerodayyy I have published the changes to the new release [v1.24.5](https://github.com/vivaxy/vscode-conventional-commits/releases/tag/v1.24.5), if you have any other questions, welcome to submit a new issue. 😊
